### PR TITLE
Update mapTo API doc

### PR DESCRIPTION
**Pull request by desistefanova**
Extends the description that `mapTo` can be user on class level.

### CODE REVIEW

File: common/lib/src/realm_common_base.dart [hunk 1]
```diff
@@ -57,10 +57,12 @@ class RealmModel {
   const RealmModel([this.type = ObjectType.realmObject]);
 }
 
-/// MapTo annotation for class member level.
+/// MapTo annotation for class level and class member level.
 ///
-/// Indicates that a property should be persisted under a different name.
-/// This is useful when opening a Realm across different bindings where code style conventions might differ.
+/// MapTo on class level indicates that a class should be persisted under a different collection name.
```
Comment (blagoev) [inline common/lib/src/realm_common_base.dart:62]:
```suggestion
/// MapTo on class level indicates that a class should be persisted under a different name.
```
The word `collection` here is not clear. I suggest removing it

File: common/lib/src/realm_common_base.dart [hunk 2]
```diff
@@ -57,10 +57,12 @@ class RealmModel {
...
+/// MapTo on class member level indicates that a property should be persisted under a different name.
```
Comment (blagoev) [inline common/lib/src/realm_common_base.dart:63]: The text now is too verbose imo. I would suggest keeping the previous text.
So 
```
/// Indicates that the class or the property should be persisted under a different name

```

**Review by blagoev — CHANGES_REQUESTED** [review]
I have some suggestions to improve the proposed changes.

File: common/lib/src/realm_common_base.dart (hunk 2 above)
Comment (desistefanova) [reply common/lib/src/realm_common_base.dart]: Ok. Done.

File: common/lib/src/realm_common_base.dart (hunk 1 above)
Comment (desistefanova) [reply common/lib/src/realm_common_base.dart]: I applied your next suggestion.

**Review by blagoev — APPROVED** [review]

**Pull request merged**